### PR TITLE
Updating bOSMO

### PR DESCRIPTION
**Pull request by tvl83**
Pending PR at chain-registry 

https://github.com/cosmos/chain-registry/pull/4614

### Upgrading Asset to Verified

If upgrading an Asset to Verified, please see the requirements specified at [LISTING](https://github.com/osmosis-labs/assetlists/blob/main/LISTING.md#upgrade-asset-to-verified-status-permissioned), and ensure the following:
- [X] Asset is defined thoroughly at the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - [x] A meaningful `description` and `extended_description`.
   - [X] Associated `socials`, including `website` and `twitter`.
   - [X] Logo Image has a square Aspect Ratio and < 250 KB file size.
- [X] This pool contains contains at least $1k USD-worth of liquidity of the asset (Provide Pool ID): 1643 ( On WhiteWhale DEX )

'Verified' Status Validation Checklist (to be completed by Osmosis Zone maintainers):
- [x] Verify appearance and metadata
- [x] Accurate Price
- [x] Trading and routing functionality
- [x] Withdraw and Deposit
   - [x] Deposit Transaction URL (if in-app)

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Do you have an extended_description?

**Comment by tvl83** [comment]
apologies I overlooked that. I would like to point out I noticed a few verified tokens don't have an extended description. is this a new requirement?

I'll add an extended description to the PR at the chain-registry asap

**Comment by JeremyParish69** [comment]
> apologies I overlooked that. I would like to point out I noticed a few verified tokens don't have an extended description. is this a new requirement?
> 
> I'll add an extended description to the PR at the chain-registry asap

Yes, it's a new-ish requirement. There are efforts to backtrack through all the assets to ensure that all tokens eventually have this.

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -3428,8 +3428,8 @@
         "liquid_staking",
         "sail_initiative"
       ],
-      "osmosis_verified": false,
-      "osmosis_unlisted": true,
+      "osmosis_verified": true,
+      "osmosis_unlisted": false,
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:3432]: We can remove the osmosis_unlisted property entirely. Actually, could you please replace it with listing date?

Comment (tvl83) [reply osmosis-1/osmosis.zone_assets.json:3432]: yes ill make this update

Comment (JeremyParish69) [reply osmosis-1/osmosis.zone_assets.json:3432]: Bumping this. We seem to have stopped right before the finish line.

Comment (tvl83) [reply osmosis-1/osmosis.zone_assets.json]: added listing timestamp and removed the `osmosis_unlisted` property

**Pull request merged**